app/views/transancts/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="category h-screen">
+<div>
+<header class="header bg-sky-600 font-bold">
+     <div>
+     <p class="text-2xl">💰</p>
+    </div>
+    <h2 class= "justify-self-center">Transanctions</h2>
+    <div>
+    <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "sign_out text_white" %>
+    </div>
+</header>
+<div class="bg py-4 px-2 text-center margin-bottom">
+   Total amount: <span class="fw-bold"> <%= number_to_currency(@category.transancts.sum(:amount)) %></span>
+</div>
+</div>
+<ul class="ul mx-auto gap-y-2 overflow-y-auto overflow-x-hidden list">
+  <% if @transancts.length == 0 %>
+         <p class="text-center text-slate-800">No transactions created yet</p>
+   <% else %>
+    <%  @transancts.each do |transanct| %>
+        <li class="list bg margin-bottom trans">
+                <div>
+                    <p class="name"><%= transanct.name %></p>
+                    <p class="smal-txt text-secondary"><%= transanct.created_at.strftime("%d %B %Y") %> at <%= transanct.created_at.strftime("%H:%M") %></p>
+                </div>                              
+                <div class="my-2">
+                     <span class="fw-bold"> <%= number_to_currency(transanct.amount) %></span>
+                </div>
+        </li>
+    <% end %>
+    <%end%>
+</ul>
+
+<div class="flex justify-center items-center bg-sky-600 text-white py-5 px-3 font-bold">
+    <%= link_to "ADD NEW TRANSANCTION", new_category_transanct_path(@category),  class:"add-button cursor-pointer hover:bg-sky-700" %>
+</div>
+</div>
```

Comment:
- [ ] I cannot return to the Home page (categories page) from the transaction page. Please add a  "Back" button (<) on the transaction page so that when the user clicks on the button the user navigates to the home page. 

![image](https://user-images.githubusercontent.com/83803414/182807445-6f26c3cd-d7d0-4e57-a408-1ef1494710f0.png)